doc/guide/cert-authentication.xml

```diff
@@ -26,19 +26,19 @@
     <para>Generating the certificates for users is usually done with a certificate management system like
       <ulink url="https://pagure.io/certmonger">certmonger</ulink> or
       <ulink url="https://www.freeipa.org/page/PKI">FreeIPA</ulink>, which are not documented here.
-      For testing purposes, these commands will generate a self-signed certificate/key for the "alice" user:</para>
+      This command generates a simple key and certificate request for the "alice" user:</para>
 
 <programlisting>
-# create self-signed certificate and key
-# some browsers insist on specifying key usage, so it needs a config file
-printf '[req]\ndistinguished_name=dn\nextensions=v3_req\n[dn]\n
-    [v3_req]\nkeyUsage=digitalSignature,keyEncipherment,keyAgreement\n' > /tmp/openssl.cnf
+openssl req -new -newkey rsa:2048 -days 365 -nodes \
```

Comment:
`-nodes`?  Raw key on disk? :(

Reply:
Nothing new, but I (locally) added a commit that adds a `shred -u alice.key` after the p12 conversion.

Reply:
I updated the commit to also drop `-nodes`. You don't need the .key for signing the CSR, and p12 conversion will happily ask about the key. This is now safe, but highly interactive -- but as this is not a course in automated key management, but just a demo how to integrate this with IdM, I think that's ok.